js/src/constants.js

```diff
@@ -8,7 +8,7 @@ const ACTIONS = {
   // actions list end here
 };
 
-const COMPOSIO_VERSION = `0.3.0`;
+const COMPOSIO_VERSION = `0.4.0-beta`;
```

Comment:
Good to see version alignment between package.json and constants.js. Consider implementing a single source of truth for version numbers to prevent future mismatches. This could be done through a version management script or using package.json as the primary source.